src/Components/TopBar.js

```diff
@@ -21,6 +21,7 @@ function TopBar() {
             <Nav.Link href="/chatroom">Chat Room</Nav.Link>
             <Nav.Link href="/sumupnum">Sum Up Number</Nav.Link>
```

Comment:
Please remove this link as it is not important according to our clients needs.

Reply:
ok, will do that.